### ISSUE TITLE
Question (or bug?): how to test/filter/exclude "None" ?

### ISSUE THREAD

**Issue by Karmak23** [datadesk/django-yamlfield#10]
Given [this model](https://github.com/1flow/1flow/blob/master/oneflow/core/models/reldb/website.py#L79) which has [a YAMLField here](https://github.com/1flow/1flow/blob/master/oneflow/core/models/reldb/website.py#L168), this fails:

```
for ws in WebSite.objects.exclude(processing_parameters=None)[:10]:
    print ws.id, ws.processing_parameters
   ....:     
28 None
48 None
74 {'wow': {'again': False, 'wiwo': 2, 'test': True, 'youbet': True, 'hehe': 1, 'encore': False}}
44 None
127 {'again': 2, 'ohmy': 3, 'omg': 1}
130 {'test': True}
6921 None
6922 None
6923 None
6924 None
```

I tried various values (`u'null'`, `u''`, even with `__in=( … )` syntax), and it always fails, eg. instances with `None` value (which are NULL in DB ?) are always returned whereas they should not.

**Comment by Karmak23** [comment]
PS: I digged in the DB, for some reason all NULL fields were in fact empty strings `''` and not the criteria works.
It seems I have a form that somewhere sets the empty string where it should leave the field alone, and empty strings are filled over time while instance are accessed. Awkward.

Sorry for the noise.

**Comment by Karmak23** [comment]
I think I found the culprit: https://github.com/datadesk/django-yamlfield/blob/master/yamlfield/fields.py#L33

It should probably be `return None` for Django to store `NULL`. I guess that returning `""` stores an empty string whereas before the field was None.

That's why my feeds get filled with `''` values over time: every time they are refreshed they are `save()`d and `None` values get replaced slowly by `''`.

What do you think about it ?

**Comment by Karmak23** [comment]
Result: with `return None` on line 33, everything works as expected on my side. Submitted a PR.

On my side I will fork the repository before the ordereddict merge until I find a fix to #9.
